src/components/TodoInput.jsx

```diff
@@ -0,0 +1,139 @@
+/** @jsxImportSource @emotion/react */
+import React, { Component } from 'react';
+import { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import { BsFillExclamationTriangleFill } from 'react-icons/bs';
+
+// 할 일 입력 form
+const inputContainer = css`
+  padding: 0 20px;
+`;
+// 할 일 입력 버튼
+const todoInputButton = css`
+  border: none;
+  background: none;
+  cursor: pointer;
+  padding: 10px;
+`;
+// 할 일 입력 창
+const inputBox = css`
+  width: 80%;
+  padding: 10px;
+  border: 1px solid lightgray;
+  border-radius: 15px;
+`;
+// Todo List 앱 제목
+const appTitle = css`
+  padding: 20px 20px 0;
+  font-style: italic;
+`;
+// hr 태그 스타일
+const line = css`
+  margin-bottom: 10px;
+  color: lightgray;
+  border-style: dashed;
+  border-width: 2px 0 0 0;
+`;
+// 할 일 입력 기능
+const TodoInput = ({ todoList, setTodoList }) => {
+  const [text, setText] = useState('');
+  const inputRef = useRef(null); // 버튼 클릭 후 input 값 초기화 후 focusing
+  const [popup, setPopup] = useState(false); // 공백 입력 방지 팝업
+  // input 값 가져오기
+  const onChangeInput = (e) => {
```

Comment:
이벤트 핸들러의 이름은 주로 `handle` + `대상` + `동작` 의 형태로 짓습니다!
```suggestion
  const handleInputChange = (e) => {
```